__tests__/token.worker.test.ts

```diff
@@ -36,10 +37,14 @@ describe('token worker', () => {
       })
     );
     const response = await messageHandlerAsync({
-      url: '/foo',
-      method: 'POST',
-      body: JSON.stringify({})
+      fetchUrl: '/foo',
+      fetchOptions: {
+        method: 'POST',
+        body: JSON.stringify({})
+      },
+      auth: {}
```

Comment:
Should we make auth optional and have it use a default value of {} (or work with undefined?)?

Reply:
Yeah makes sense, could be feasible that you don't pass any audience or scope in. I've made it optional on the type and modified the test.